### PR TITLE
Nuget badges in readme.md

### PR DESCRIPTION
**Pull request by Kielek**
Towards #906

## Changes

Nuget badges in all readme.md files.

For significant contributions please make sure you have completed the following items:

* ~~[ ] Appropriate `CHANGELOG.md` updated for non-trivial changes~~
* ~~[ ] Design discussion issue #~~
* ~~[ ] Changes in public API reviewed~~

### CODE REVIEW

**Review by xsoheilalizadeh — APPROVED** [review]

File: src/OpenTelemetry.Instrumentation.AWSLambda/README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # AWS OTel .NET SDK for Lambda
 
+[![nuget](https://img.shields.io/nuget/v/OpenTelemetry.Instrumentation.AWSLambda.svg)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.AWSLambda)
```
Comment (Oberon00) [inline src/OpenTelemetry.Instrumentation.AWSLambda/README.md:3]:
```suggestion
[![nuget version badge](https://img.shields.io/nuget/v/OpenTelemetry.Instrumentation.AWSLambda.svg)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.AWSLambda#versions-body-tab)
```

File: src/OpenTelemetry.Instrumentation.AWSLambda/README.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+[![NuGet](https://img.shields.io/nuget/dt/OpenTelemetry.Instrumentation.AWSLambda.svg)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.AWSLambda)
```
Comment (Oberon00) [inline src/OpenTelemetry.Instrumentation.AWSLambda/README.md:4]:
```suggestion
[![NuGet download count badge](https://img.shields.io/nuget/dt/OpenTelemetry.Instrumentation.AWSLambda.svg)](https://www.nuget.org/stats/packages/OpenTelemetry.Instrumentation.AWSLambda?groupby=Version)
```

File: src/OpenTelemetry.ResourceDetectors.AWS/README.md [hunk 3]
```diff
@@ -1,5 +1,8 @@
 # AWS Resource Detectors
 
+[![nuget](https://img.shields.io/nuget/v/OpenTelemetry.ResourceDetectors.AWS.svg)](https://www.nuget.org/packages/OpenTelemetry.ResourceDetectors.AWS)
```
Comment (xsoheilalizadeh) [inline src/OpenTelemetry.ResourceDetectors.AWS/README.md:3]: It couldn't find the package, is it expected?

**Review by Oberon00 — APPROVED** [review]
The URLs are not at all self-explanatory, so I was wondering for a moment why we have the same badge two times.

File: src/OpenTelemetry.ResourceDetectors.AWS/README.md (hunk 3 above)
Comment (Kielek) [reply src/OpenTelemetry.ResourceDetectors.AWS/README.md:3]: This package was not yet released. It is expected.

File: src/OpenTelemetry.Instrumentation.AWSLambda/README.md (hunk 2 above)
Comment (Kielek) [reply src/OpenTelemetry.Instrumentation.AWSLambda/README.md:4]: @Oberon00 - what d o you think about this link https://www.nuget.org/packages/OpenTelemetry.Instrumentation.AWSLambda#versions-body-tab?

Serves similar data, but I think there is easier way to find other tabs

Comment (Oberon00) [reply src/OpenTelemetry.Instrumentation.AWSLambda/README.md:4]: Probably it's also fine to keep the originally proposed links. I have no strong opinion here.

Comment (Kielek) [reply src/OpenTelemetry.Instrumentation.AWSLambda/README.md:4]: I will change descriptions and keep links.

**Review by fred2u — APPROVED** [review]

**Review by rypdal — APPROVED** [review]

**Review by Oberon00 — APPROVED** [review]

**Review by reyang — APPROVED** [review]

**Review by utpilla — APPROVED** [review]

**Pull request merged**